### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by stdavis** [agrc/wfrc-rtp-projects#11]

**Comment by nathankota** [comment]
🤔 @stdavis can  you please remind me how that works on [the wasatch choice app](https://wfrc.org/wasatch-choice-map/#mapList=vision.transportation.landuse.econdev.recreation&selectedMap=vision&sideBarClosed=false)?

**Comment by stdavis** [comment]
The [use the config](https://github.com/agrc/wfrc-wasatch-choice#internationalization) to manage translations.

**Comment by nathankota** [comment]
Yes, they are interested in internationalization, but want to know if it is significantly more effort, and what is needed from them to implement.

**Comment by stdavis** [comment]
Now that I have a system to emulate from the wasatch choice project, I think that it would only be 2-4 hours. I would just need the translations for everything that they want to be translated.

**Comment by nathankota** [comment]
Client will provide translations.
